README.md

```diff
@@ -31,7 +31,9 @@ commercial distribution of Janssen Project Components called
 
 **Contribute**: [Contribution Guide](https://docs.jans.io/head/CONTRIBUTING/) | [Community Docs](https://docs.jans.io/head/governance/charter/) | [Developer Guides](https://docs.jans.io/head/CODE_OF_CONDUCT/)
 
-**Social**: [Linkedin](https://www.linkedin.com/company/janssen-project)
+**Social**: [Linkedin](https://www.linkedin.com/company/janssen-project) | [YouTube](https://www.youtube.com/@JanssenProject)
+
+**Learn**: [Gluu Academy](https://learn.gluu.org/)
```

Comment:
Can you give this a title before it. Maybe resources to learn more

Just because we have to have a clear distinction when adding Gluu in Janssen

Reply:
Updated the title :+1: